composer.json

```diff
@@ -71,53 +71,51 @@
       "@lint:dominant-color-images",
       "@lint:embed-optimizer",
       "@lint:optimization-detective",
+      "@lint:performance-lab",
       "@lint:speculation-rules",
       "@lint:webp-uploads"
     ],
     "lint:auto-sizes": "@lint -- ./plugins/auto-sizes --standard=./plugins/auto-sizes/phpcs.xml.dist",
     "lint:dominant-color-images": "@lint -- ./plugins/dominant-color-images --standard=./plugins/dominant-color-images/phpcs.xml.dist",
     "lint:embed-optimizer": "@lint -- ./plugins/embed-optimizer --standard=./plugins/embed-optimizer/phpcs.xml.dist",
     "lint:optimization-detective": "@lint -- ./plugins/optimization-detective --standard=./plugins/optimization-detective/phpcs.xml.dist",
+    "lint:performance-lab": "@lint -- ./plugins/performance-lab --standard=./plugins/performance-lab/phpcs.xml.dist",
     "lint:speculation-rules": "@lint -- ./plugins/speculation-rules --standard=./plugins/speculation-rules/phpcs.xml.dist",
     "lint:webp-uploads": "@lint -- ./plugins/webp-uploads --standard=./plugins/webp-uploads/phpcs.xml.dist",
     "phpstan": "phpstan analyse --memory-limit=2048M",
-    "test": "phpunit --verbose --testsuite performance-lab",
-    "test-multisite": "phpunit -c tests/multisite.xml --verbose --testsuite performance-lab",
-    "test-multisite:all": [
-      "@test-multisite",
-      "@test-multisite:plugins"
-    ],
-    "test-multisite:auto-sizes": "phpunit -c tests/multisite.xml --verbose --testsuite auto-sizes",
-    "test-multisite:dominant-color-images": "phpunit -c tests/multisite.xml --verbose --testsuite dominant-color-images",
-    "test-multisite:embed-optimizer": "phpunit -c tests/multisite.xml --verbose --testsuite embed-optimizer",
-    "test-multisite:optimization-detective": "phpunit -c tests/multisite.xml --verbose --testsuite optimization-detective",
+    "test": "phpunit",
```

Comment:
`--verbose` is intentionally removed? I suppose there's no need for it to be supplied here, as it can be provided when actually invoking. Perhaps it should then be added in the workflow?

Reply:
Oh, nevermind. It _is_ still present below. Would anyone call `composer run-script test` directly then?

Reply:
It's being called in:

https://github.com/WordPress/performance/blob/d9cd4d4d6e47de59e301228399c8fe8e400cf873/package.json#L46-L47

In case someone wants to use binary their way.